bodhi/server/util.py

```diff
@@ -437,7 +437,7 @@ def type2icon(context, kind):
         'enhancement': 'fa-bolt',
     }.get(kind)
 
-    return "<span class='label label-%s'><i class='fa %s'></i></span>" % (cls, fontawesome)
+    return "<span class='label label-%s' data-toggle='tooltip' title='%s update'><i class='fa %s'></i></span>" % (cls, kind, fontawesome)
```

Comment:
Since the critical path one is more of a complete sentence, I suggest either making this one also be a complete sentence, or making the critical path one just say "critical path". For example, this title bit could be:

```
title='This is a %s update'
```